client/ladder/ladder.tsx

```diff
@@ -202,6 +204,16 @@ export function LadderTable(props: LadderTableProps) {
   const renderNumber = useCallback<TableCellRenderer>(props => {
     return <NumberText>{props.cellData}</NumberText>
   }, [])
+  const renderStats = useCallback<TableCellRenderer>(props => {
+    return (
+      <NumberText>
+        {props.cellData.wins} &ndash; {props.cellData.losses}
+      </NumberText>
+    )
+  }, [])
+  const renderLastPlayed = useCallback<TableCellRenderer>(props => {
+    return <NumberText>{timeAgo(Math.floor((nowRef.current - props.cellData) / 1000))}</NumberText>
```

Comment:
Honestly I find any API that takes seconds to be pretty weird in JS, given that Dates operate on millis natively. I'd move this division back into the timeAgo where it was previously